misc/python/materialize/mzcompose.py

```diff
@@ -1146,16 +1147,13 @@ def __init__(
 
         entrypoint.append(f"--default-timeout {default_timeout}")
 
-        if volumes_extra:
-            volumes.extend(volumes_extra)
-
         if seed:
             entrypoint.append(f"--seed {seed}")
 
         if shell_eval:
             # Evaluate the arguments as a shell command
             # This allows bashisms to be used to prepare the list of tests to run
-            entrypoint.append("`${TD_TEST:-$$*}`")
+            entrypoint.append("${TD_TEST:-`$$*`}")
```

Comment:
this confuses me a bit, if people don't specify TD_TEST, what is actually being expanded? the arguments after the `mzcompose run testdrive`?

Reply:
It confuses me too, but the tl;dr is that Bash monstrosity below makes it so that `$$*` expands to the command passed in here: https://github.com/MaterializeInc/materialize/blob/7434a8f5d21dadbed47df3e77e756ddbd6915a6d/test/redpanda/mzworkflows.py#L34

Reply:
Which, yeah, can be overridden if you specify args after `./mzcompose run testdrive`.